libs/dh/metering-point/feature-overview/src/components/dh-metering-point.component.ts

```diff
@@ -233,8 +233,6 @@ export class DhMeteringPointComponent {
 
   constructor() {
     effect(() => {
-      this.breadcrumbService.navigationEnded();
-
```

Comment:
suggestion: looks like this was intentionally added https://github.com/Energinet-DataHub/greenforce-frontend/pull/4167/files Can you test if everything still works after removing it? Like navigating between parent -> child -> parent metering points.